src/net/net_to_hvmc.rs

```diff
@@ -54,7 +54,7 @@ fn net_tree_to_hvmc_tree(
       rgt: Box::new(var_or_subtree(inet, Port(tree_root, 2), port_to_var_id, id_to_hvmc_name)),
     },
     NodeKind::Con { lab: Some(lab) } => Tree::Dup {
-      lab: (lab + 1) << 1 | 0,
+      lab: (lab + 1) << 1,
```

Comment:
Can you keep this one? So that it pairs correctly with the dup node below.
Also, it's like "label plus bit 0 is set to 0" so I think how it was before conveys that better.